### DISCUSSION TITLE
Setting up not to use cache

### DISCUSSION THREAD

**Discussion by hgjo-nanum** [vercel/next.js#70445] [Help]
### Summary

Hello! I have a question.
Is there a setting for nextjs to disable cache itself?
There are too many caches left in the .next/cache/fetch-cache folder, which is a burden on disk space.
Is there a setting to disable the caching function so that no cache remains?

**Comment by kimpuro** [comment]
I think the way to go would be to use the no-cache option in the fetch request.
Below is the explanation from the Next.js doc:
By setting the cache option to no-store, we can indicate that the fetched data should [never be cached](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching). This is similar to getServerSideProps in the pages directory.

```async function getProjects() {
  const res = await fetch(`https://...`, { cache: 'no-store' })
  const projects = await res.json()
 
  return projects
}
 
export default async function Dashboard() {
  const projects = await getProjects()
 
  return (
    <ul>
      {projects.map((project) => (
        <li key={project.id}>{project.name}</li>
      ))}
    </ul>
  )
}```

**Reply by hgjo-nanum** [reply]
Thank you for your reply.
My question was an extension of the post below.
https://github.com/vercel/next.js/discussions/70387

I was requesting data using the supabase client, and I was worried that there would be too many pages if I made an API page and made a fetch request.
I resolved it by defining an option when requesting from the supabase client by referring to the post below.
https://github.com/orgs/supabase/discussions/20022#discussioncomment-9757485

**Reply by kimpuro** [reply]
I'm glad your issue has been resolved.
Did using the no-store option in the fetch request that was returned help you out?
